src/main/js/api/sse.js

```diff
@@ -52,22 +50,22 @@ exports.connect = function(browser, done) {
  */
 exports.disconnect = function(onDisconnected) {
     function clientDisconnect() {
-        sseClient.disconnect();
+        if (sseConnection) {
+            sseConnection.disconnect();
```

Comment:
That would be an SSE gateway change. We haven't really needed it so it was never added. Not even sure you can get a close event on the channel, but I'm guessing you prob can (I forget).